site/.vitepress/config.ts

```diff
@@ -503,13 +494,8 @@ export default defineConfig({
             text: "Utils",
             collapsed: true,
             base: "/packages/aa-alchemy/utils",
-            items: [
-              {
-                text: "Overview of Utils",
-                link: "/introduction",
-              },
-              { text: "SupportedChains", link: "/supportedChains" },
-            ],
```

Comment:
let's put this back, per my comment re: no section header pages to make each page explicitly visible on sidebar